usecases/backup/handler.go

```diff
@@ -49,7 +49,7 @@ type schemaManger interface {
 	NodeName() string
 }
 
-type nodeResolver interface {
+type NodeResolver interface {
```

Comment:
**Question:** why is this interface now publically exported? I can't seem to find any instances of `backup.NodeResolver` in any of the diff 🤔

Reply:
that's exported to be able to test atm, this interface actually shall be removed not needed, but no time to do this refactor now